src/components/frame/Extensions/pages/partials/ActiveInstalls.tsx

```diff
@@ -1,35 +1,25 @@
 import React, { FC } from 'react';
-import { useIntl } from 'react-intl';
 
 import { MAX_INSTALLED_NUMBER } from '~constants';
 import { ActiveInstallsProps } from './types';
 import ExtensionStatusBadge from '~v5/common/Pills/ExtensionStatusBadge';
-import { SpinnerLoader } from '~shared/Preloaders';
 import { formatText } from '~utils/intl';
 
 const displayName = 'frame.Extensions.pages.partials.ActiveInstalls';
 
 const ActiveInstalls: FC<ActiveInstallsProps> = ({ activeInstalls }) => {
-  const { formatMessage } = useIntl();
-
   return (
     <div className="sm:hidden md:block">
-      {activeInstalls ? (
-        <>
-          {activeInstalls < MAX_INSTALLED_NUMBER ? (
-            <ExtensionStatusBadge
-              mode="new"
-              text={formatText({ id: 'status.new' })}
-            />
-          ) : (
-            <p className="text-gray-400 text-sm text-right">
-              {activeInstalls.toLocaleString('en-US')}{' '}
-              {formatMessage({ id: 'active.installs' })}
-            </p>
-          )}
-        </>
+      {activeInstalls < MAX_INSTALLED_NUMBER ? (
```

Comment:
If number of `activeInstalls` is a below this constant (currently 39 (?) ), it will display a "new" status pill, else it'll show the actual count.

Reply:
Could we rename this const to something more meaningful?